src/util.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * __________ In/Out Util __________
+ */
+function biasedRandom(min, max, bias) {
+  if (bias === -1) {
+    return Math.ceil((1 - Math.cbrt(1 - Math.random())) * (max - min));
+  } else if (bias === 1) {
+    return Math.floor(Math.sqrt(Math.sqrt(1 - Math.random())) * (max - min));
+  } else {
+    return Math.floor(Math.random() * (max - min));
+  }
+}
+
+function timeGroup(hour) {
+  if (hour < 6 || hour > 20) {
```

Comment:
I was going to say that this is redundant since the coffee shop backend takes care of hours. But then I realized that if for some reason they were to query during these hours, it makes a lot of sense to have the shop be closed (ie. no customers in or out). I like this.

One comment about this though is that I think Starbucks actually closes at 9pm, so the hours may have to be adjusted.

Reply:
Adjusting shouldn't be a problem, but we can have a team discussion about whether or not it's worth expecting stores to send their own hours in the request body. It wouldn't be too hard but might be a little unnecessarily robust for the purposes of the API. We can see what everyone thinks.